snippets/mcs-endpoint-limitation.adoc

```diff
@@ -0,0 +1,14 @@
+// Text snippet included in the following modules:
+//
+// * modules/installation-about-custom-azure-vnet.adoc
+// * modules/machine-config-operator.adoc
+// * security/certificate_types_descriptions/machine-config-operator-certificates.adoc
+
+:_content-type: SNIPPET
+
+[IMPORTANT]
+====
+There is no supported way to block or restrict the Machine Config Server (MCS) endpoint in the current architecture. The MCS must be exposed to the network so that newly-provisioned machines, which have no existing configuration or state, are able to fetch their configuration. In this model, the root of trust is the certificate signing requests (CSR) endpoint, which is where the kubelet sends its certificate signing request for approval to join the cluster. Because of this, machine configs should not be used to distribute sensitive information.
```

Comment:
* No hyphen for adverbs ending in -ly
* Also, does Machine Config Server need to be capitalized? One of the files has it lowercase elsewhere in the file.

```suggestion
There is no supported way to block or restrict the Machine Config Server (MCS) endpoint in the current architecture. The MCS must be exposed to the network so that newly provisioned machines, which have no existing configuration or state, are able to fetch their configuration. In this model, the root of trust is the certificate signing requests (CSR) endpoint, which is where the kubelet sends its certificate signing request for approval to join the cluster. Because of this, machine configs should not be used to distribute sensitive information.
```

Reply:
TIL!

Reply:
> does Machine Config Server need to be capitalized

The term is capitalized in several ways throughout the docs (!). I must have picked up the initial caps from the instance in [machine-config-operator-certificates.adoc]( https://github.com/openshift/openshift-docs/commit/2922f04b5f3f19e8865925484d441bcca238cc1b#diff-b49ae0cef19e9f71a19a52b64dbca547f734b93e3f3085c68204e0a65a1402c4R11).

I can make it lower case in these two places.

Reply:
Good to be consistent, just in case